.gitattributes

```diff
@@ -1,3 +1,4 @@
 * text=auto
 *.js text eol=lf
-*.ts text eol=lf
\ No newline at end of file
+*.ts text eol=lf
+*.lint eol=lf
```

Comment:
Required to make multi-line lint errors pass the tests. https://github.com/palantir/tslint/issues/4171